src/main/scala/scalismo/statisticalmodel/SVIMCoefficients.scala

```diff
@@ -0,0 +1,27 @@
+package scalismo.statisticalmodel
+
+import breeze.linalg.DenseVector
+
+case class SVIMCoefficients(shape: DenseVector[Double],
+    intensity: DenseVector[Double]) {
+  def *(f: Float): SVIMCoefficients = this * f.toDouble
+  def *(d: Double): SVIMCoefficients = copy(shape = shape * d, intensity = intensity * d)
+  def +(other: SVIMCoefficients): SVIMCoefficients = copy(shape = shape + other.shape, intensity = intensity + other.intensity)
+  def -(other: SVIMCoefficients): SVIMCoefficients = copy(shape = shape - other.shape, intensity = intensity - other.intensity)
+}
+
+object SVIMCoefficients {
+  def apply(shape: IndexedSeq[Double],
+    color: IndexedSeq[Double]) = new SVIMCoefficients(DenseVector(shape.toArray), DenseVector(color.toArray))
+
+  def apply(shape: DenseVector[Double],
+    color: DenseVector[Double]) = new SVIMCoefficients(shape, color)
```

Comment:
to be consistent, replace "color" with "intensity"

Reply:
I agree, this is still a left-over from the MoMo class I took as a template.